setup/src/main/java/aero/minova/cas/setup/xml/table/ColumnBigint.java

```diff
@@ -0,0 +1,16 @@
+package aero.minova.cas.setup.xml.table;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ColumnBigint {
+    private boolean nullable = true;
+
+    public boolean getNullable() {
```

Comment:
Du hast doch die @Getter und @Setter Annotationen. Für was brauchst du dann noch die Methode?

(Dasselbe in den anderne Column*-Klassen)

Reply:
Man hätte zu den Column*-Klassen bestimmt auch ein Interface machen können.

Es haben ja fast alle private boolean nullable = true und private boolean identity.

Reply:
> Du hast doch die @getter und @Setter Annotationen. Für was brauchst du dann noch die Methode?
> 
> (Dasselbe in den anderne Column*-Klassen)

Ich wollte in dem originalen Install-Tool so wenig wie möglich ändern (irony).
@Getter erzeugt aber die Methode is Nullable() und im Original heisst es getNullable().

Reply:
> Man hätte zu den Column*-Klassen bestimmt auch ein Interface machen können.
> 
> Es haben ja fast alle private boolean nullable = true und private boolean identity.

Habe keine Vererbung angewandt,
da ich erstmal nur stumpf die Klassen (bzw. das XML-Format) übersetzen wollte und keine neuen Hierarchien einführen wollte.
Dadurch vermeidet man automatisch Fehler bei der Portierung, für denn Fall,
dass eine Hierachie nicht so funktioniert, wie man sich das am Anfang gedacht hat.

Man kann natürlich dafür eine gemeinsame Basisklasse noch einführen.